AphroCite.h

```diff
@@ -62,12 +84,94 @@ void Test_CheckCondition(bool32 condition, char *description){
     }
 }
 
-void Assert_True(char* description, bool32 condition){
+void Assert_True(CString description, Boolean32 condition){
     Test_CheckCondition(condition, description);
 }
 
-void Assert_Compare(char* description, void* actual, void* desired, int size){
+void Assert_Compare(CString description, void* actual, void* desired, int size){
     Test_CheckCondition(Memory_Compare(actual, desired, size) == 0, description);
 }
 
+/*
+    Unit Testing Functionallity
+*/
+
+typedef struct _T_UnitTest_TestSuite {
+    CString name;
+    Boolean32 silentMode;
+
+    int testCount;
+    int failedCount;
+    int succeededCount;
+} T_UnitTest_TestSuite;
```

Comment:
Yes because then it is clear what is is, it is just my convention that i picked up but we can remove it if you would prefere that